### ISSUE TITLE
Turn off pollution on officials until it gets reworked

### ISSUE THREAD

**Issue by Sphyix** [GTNewHorizons/GT-New-Horizons-Modpack#7617]
Right now there's no way to deal with pollution..
Pollution scrubbers are buggy, and the only way to be able to stay in your base without an hazmat is by spreading it enough so every chunk is kept under 500k (which is REEEEEALLY easy to get to)
I think that the fact that pollution is not adding literally anything to the gameplay right now has already been said multiple times, but I guess I'll mention it aswell, pollution is just annoying and doesn't add anything interesting to the gameplay, except making sure you spread your base enough (which is kinda bad, I mean, long cable runs cause lag, and power needs to be centralized for efficiency most of the times)
So the request is to turn off pollution on official servers while waiting for the next update, as right now it's just an annoyance causing ppl to change the way they build their bases that could cause issues in the long run (like mentioned above, the power part)

tl;dr pollution is bad, base is smol, turn off pollution so I don't choke

**Comment by basdxz** [comment]
Wow, amazing, absolutely astonishing. Duplicate of #7341 But still needs attention.

**Comment by Sphyix** [comment]
Well, I mean, it's saying the same things, but I'm asking to temporarily turn it off on officials

**Comment by basdxz** [comment]
Rework has been stagnant. Too many cooks in the kitchen and no drive to get anything done.

**Comment by Prometheus0000** [comment]
Didn't DvDmanDT say they were going to work on it?

**Comment by Sphyix** [comment]
> Rework has been stagnant. Too many cooks in the kitchen and no drive to get anything done.

Then the request to turn it off until it's reworked is more than valid I think?
At least even if they never rework it, we can play without something ruining the experience

**Comment by boubou19** [comment]
@Sphyix iirc, pollution option only disable the creation of pollution, but chunks stay polluted. So officials are doomed until it's either reworked or disabled in the next version the officials will receive.

**Comment by dvdmandt** [comment]
For the record, I am in early stages of revamping this indeed, but my work is more or less a ground up redesign that is not going to be done in an afternoon (think months). Also, the stuff I'm working on hasn't really been generally approved or anything, so there isn't even a guarantee that it'll be merged/included in the pack even when implemented.

So we may need some quick fixes for now if the current situation is unbearable for a significant portion of the player base. If not a disable, perhaps we can speed up dissipation and/or adjust the effect limits to be less destructive and annoying for now?

**Comment by Leche-Caliente** [comment]
@boubou19 not my experience on my personal world. I had pollution on until I started getting really bad constant status effects, then I turned it off in config. The chunks still read as polluted on my scanner, but no status effects at all.

**Comment by z3r0b0t** [comment]
I would really like to see this happening, 

I've been playing for almost 1 year now, I experienced pollution once in LV. I'm progressing very slowly on officials, mainly because that's how I play. I plan everything in advance (sometimes very poorly), and for me pollution is just an obstacle for my base build and its planning (spoiler: I wanna go vertical).

I hear really crazy stories about it and don't know what to expect , but I understand it will be a pain. I gathered a few points of why I think it should be disabled asap on officials : 
- It has a negative impact on how players could get creative with their builds, in a game that's all about that I find that mucho sad. I read way too often something along the lines of "you should spread your base because **pollution**"
- Also, spreading your base seems like a band-aid solution to me, more chunks will be loaded, potentially more lag and makes it more expensive to progress (need more cable/pipes/diode/chunkloaders/etc.)
- Is a really bad mechanic, I assume it was to prevent some sort of abuse but the reason behind it seems to have been lost or is unclear to me.
- Could be a really great mechanic with a good rework, but in its current state is just a pain in the booty, hence why it should be turned off asap
- IMO, it should in no way arm the terrain _permanently_
- A.K.A the current state of pollution is destructive, way more than a simple machine blowing up because of rain; I can easily see how that could remove some fun in the game for some people.

Please be gentle, I am still a newbie at this modpack, I haven't experienced pollution at higher tiers and I'm talking mainly based on the one experience I had, as a new player and from the fear that has grown in me after a few discussions about pollution.

If this issue/proposition is not going anywhere, I would really like to see this transformed into a petition and share it to players on officials, to see if anyone else (and how many) have a similar point of view and get things moving with whoever it may concern afterwards. I just have a feeling I am not alone feeling this way towards pollution.
Also, I really think this would create a really good opportunity for the devs to take their time to rework it properly.

Most of my points are based on these revamp visions : 
https://gtnh.miraheze.org/wiki/Pollution_revamp_visions
https://gtnh.miraheze.org/wiki/Pollution_revamp
If you haven't seen them, I suggest to take a look, there are lots of great ideas in there, but most of them agree that : 
### Pollution is annoying, it is a bad mechanic for non-issue and should be less destructive.

**Comment by MadeofGold** [comment]
Considering the fact that big pollution shouldn't be an issue untill HV, you could just make the carbon nanosuit (all pieces give full hazmat) or Quantum suit ( full hazmat)
